### ISSUE TITLE
Admin Left Navigation present in Email Template preview window

### ISSUE THREAD

**Issue by vasilii-b** [magento/magento2#7259]
### Preconditions

1. Magento 2.1, upgraded from 2.0
2. __OR Magento 2.2.0-dev__

### Steps to reproduce

1. Go to Email Templates
2. Create new email template ( my example New Order)
3. Go to Email Templates Grid
4. Press 'Preview'

### Expected result

1. Preview windows should appear with email template html, *without left navigation*

### Actual result

1. In preview window there is *left navigation present*
[Screenshot](https://cloud.githubusercontent.com/assets/13456702/19888755/5fe85096-a039-11e6-989c-c8855e8cefff.jpg)

**Comment by dthampy** [comment]
Internal Jira Ticket to track this issue : MAGETWO-60448

**Comment by vasilii-b** [comment]
@dthampy  Got the fix for this issue
File `app\code\Magento\Email\view\adminhtml\layout\adminhtml_email_template_preview.xml`

``` xml
<?xml version="1.0"?>
<!--
/**
 * Copyright © 2016 Magento. All rights reserved.
 * See COPYING.txt for license details.
 */
-->
<page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" layout="admin-1column" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
    <update handle="empty" />
    <body>
        <referenceContainer name="root">
            <block name="preview.page.content" class="Magento\Framework\View\Element\Template" template="Magento_Email::template/preview.phtml">
                <block class="Magento\Email\Block\Adminhtml\Template\Preview" name="content" as="content"/>
            </block>
        </referenceContainer>
    </body>
</page>

```

**Comment by magento-engcom-team** [comment]
@vasilii-b, thank you for your report.
The issue is already fixed in develop branch, 2.2.0
